src/plot.cpp

```diff
@@ -39,4 +973,12 @@ voxel_finalize(hid_t dspace, hid_t dset, hid_t memspace)
   H5Sclose(memspace);
 }
 
+RGBColor rdm_color() {
+  RGBColor rgb;
+  rgb[RED] = int(prn()*255);
+  rgb[GREEN] = int(prn()*255);
+  rgb[BLUE] = int(prn()*255);
+  return rgb;
```

Comment:
I did this when when creating colors before with
`RGBColor(int(prn()*255), int(prn()*255), int(prn()*255))`
and it caused the plot colors to change and existing tests to fail.

You suggestion doesn't, however. Interesting. At any rate, not doing that was solely to avoid changes in the test results - not a good reason to avoid it (in this case). Will update.